frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -269,6 +269,64 @@ function InputContainer:onGesture(ev)
     end
 end
 
+-- Will be overloaded by the Gestures plugin, if enabled, for use in _onGestureFiltered
+function InputContainer:isGestureAlwaysActive(ges, multiswipe_directions)
+    -- If the plugin isn't enabled, block everything, we'll restore standard behavior on resume.
+    return false
```

Comment:
"block everything" ? shouldn't everything work if there is no way to restore the feature ?
"we'll restore standard behavior on resume." I'm a bit confused about thow this is related to suspend/resume. Or do you mean "on restore" ?

Reply:
Resume restores the standard handler, so it makes sense to actually block every gesture, even without the Gesture plugin enabled. i.e., you could trigger this via Dispatcher somehow (Profiles? QuickMenu?), and just rely on the fact that once you suspend the device, everything will behave on resume as it restores gesture handling (this is apparently how this works on Kindles without buttons).

Reply:
OK. Best to tell this somehow in the comment: "If the plugin isn't enabled, and invoked from Profiles or QuickMenu, still honor the request and block everything. The user can get out of it by suspending and resuming with the power button" (if I get what you mean right :) need to suspend before resuming. And this is not involved in a normal Suspend/Resume)